### ISSUE TITLE
Services - which classes?

### ISSUE THREAD

**Issue by RoySegall** [amitaibu/og#160]
I'm still working on the entity access issue and it's look to me that `OgGroupAudienceHelper` should be defined as a service, no? Maybe projects like OpenScholar or Drupal Commons would like to swap the class with their own implementation.

And then I was thinking - should we replace all the classes into services? 

Thoughts?

@chx @amitaibu @pfrenssen @damiankloip ^^

**Comment by amitaibu** [comment]
Yeah, the idea is that we start as simple hardcoded classes, and then after things stabilise will expose them as services

**Comment by pfrenssen** [comment]
I agree, we can convert most of the functionality that is currently in public static methods into services. Maybe we can keep a number of public static methods in `Og` for commonly used things so they can be used in procedural code such as hook implementations, but they should then be simple wrappers around the service calls.
